disassembler.html

```diff
@@ -246,7 +246,8 @@
 			var coords = mirror.charCoords({ line: highlighted_line, ch: 0 }, "local");
 			
 			mirror.addLineClass(highlighted_line, "background", "highlighted");
-			mirror.scrollTo(0, coords.y - 30);
+            console.log(coords);
```

Comment:
The whitespace here (as well as on the console.log above) looks funky. Would you please fix?